manifest.mf

```diff
@@ -2,5 +2,5 @@ Manifest-Version: 1.0
 OpenIDE-Module: org.netbeans.modules.php.wordpress
 OpenIDE-Module-Layer: org/netbeans/modules/php/wordpress/resources/layer.xml
 OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/php/wordpress/Bundle.properties
-OpenIDE-Module-Specification-Version: 0.9.5
+OpenIDE-Module-Specification-Version: 0.9.6
```

Comment:
Don't change this.

Reply:
Should I reset/revert it or make a new commit?

Reply:
Choose the way you can.